lib/fulfillment_resolver.js

```diff
@@ -0,0 +1,58 @@
+const { isInRecap } = require('./util')
+
+class FulfillmentResolver {
+  constructor (responseReceived) {
+    this.elasticSearchResponse = responseReceived
+  }
+  responseWithFulfillment () {
+    for (let hit of this.elasticSearchResponse.hits.hits) {
+      (hit._source.items || []).map((item) => {
+        if (item.electronicLocator) return item
+        const _isInRecap = isInRecap(item)
+        item.physFulfillment = {'@id': this._determinePhysFulfillment(item, _isInRecap, false)}
```

Comment:
If `_determinePhysFulfillment` returns nothing, this results in a broken statement ( `..., "physFulfillment": {}, ...`). If `_determinePhysFulfillment` returns nothing, we should not set the fulfillment property

Reply:
updated!